### DISCUSSION TITLE
Where is Changelog

### DISCUSSION THREAD

**Discussion by sysmat** [qos-ch/logback#668] [Q&A]
- Why you don't have Changelog?
- on github you could have `Changelog.md` 

```
# Version

## Features

## Fixes

## Breaking changes

## Dependencies
```

**Comment by gaeljw** [comment]
Would love to have a CHANGELOG in the repo as well to be able to see it with Renovate or Dependabot PRs for instance.

Especially when patch releases bring unexpected breaking changes.

That being said, the closest to a Changelog is the page https://logback.qos.ch/news.html
